CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [Unreleased]
+
+### Added
+
+- Notification service 
```

Comment:
Let's have it simpler, like "Notifications for CCD and CIS-2 token transactions"